service/src/test/java/uk/gov/hmcts/reform/fpl/service/CreateC21OrderServiceTest.java

```diff
@@ -0,0 +1,174 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.C21Order;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.C21OrderBundle;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+
+import java.time.LocalDateTime;
+import java.time.format.FormatStyle;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.HIS_HONOUR_JUDGE;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createPopulatedChildren;
+
+@ExtendWith(SpringExtension.class)
+class CreateC21OrderServiceTest {
+    private static final String LOCAL_AUTHORITY_CODE = "example";
+    private static final String COURT_NAME = "Example Court";
+    private static final String COURT_EMAIL = "example@court.com";
+    private static final String CONFIG = String.format("%s=>%s:%s", LOCAL_AUTHORITY_CODE, COURT_NAME, COURT_EMAIL);
+    private static final LocalDateTime NOW = LocalDateTime.now();
+
+    private final Time time = () -> NOW;
+
+    private DateFormatterService dateFormatterService = new DateFormatterService();
+    private HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration = new HmctsCourtLookupConfiguration(CONFIG);
+    private CreateC21OrderService createC21OrderService = new CreateC21OrderService(dateFormatterService,
```

Comment:
```suggestion
    private CreateC21OrderService service = new CreateC21OrderService(dateFormatterService,
```

Reply:
we can do this because in this context the service is defined in the class of the unit test